packages/queued-request-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/queued-request-controller",
-  "version": "0.6.1",
+  "version": "0.6.2",
```

Comment:
Should this be bumped to 0.7.0 since the peer dependency has been bumped?

Reply:
😶‍🌫️ yes. Actually surprised this wasn't caught by some validation since we're bumping it in the changelog... but I suppose that's a more complex action

Reply:
done here: https://github.com/MetaMask/core/pull/4111/commits/d4482d307946dc8eae05cea2992ce31f166e6bb6